# originated from https://git.alpinelinux.org/aports/plain/main/openldap/0004-Fix-manpages.patch

From 08334cb28550230ab73aa1bc60ad4f62d6f85486 Mon Sep 17 00:00:00 2001
From: Lars Kellogg-Stedman <lars@oddbit.com>
Date: Fri, 19 Aug 2022 22:54:43 -0400
Subject: [PATCH 4/8] Fix manpages

Various manual pages changes:
* removes LIBEXECDIR from slapd.8
* removes references to non-existing manpages (bz 624616)

Patch-Source: https://src.fedoraproject.org/rpms/openldap/blob/f27/f/openldap-manpages.patch
---
 doc/man/man1/ldapmodify.1 | 3 +--
 doc/man/man8/slapd.8      | 8 ++++----
 2 files changed, 5 insertions(+), 6 deletions(-)

diff --git a/doc/man/man1/ldapmodify.1 b/doc/man/man1/ldapmodify.1
index 1104e9f..161f2ff 100644
--- a/doc/man/man1/ldapmodify.1
+++ b/doc/man/man1/ldapmodify.1
@@ -382,8 +382,7 @@ exit status and a diagnostic message being written to standard error.
 .BR ldap_add_ext (3),
 .BR ldap_delete_ext (3),
 .BR ldap_modify_ext (3),
-.BR ldap_modrdn_ext (3),
-.BR ldif (5).
+.BR ldif (5)
 .SH AUTHOR
 The OpenLDAP Project <http://www.openldap.org/>
 .SH ACKNOWLEDGEMENTS
diff --git a/doc/man/man8/slapd.8 b/doc/man/man8/slapd.8
index a93fcbc..1e64855 100644
--- a/doc/man/man8/slapd.8
+++ b/doc/man/man8/slapd.8
@@ -5,7 +5,7 @@
 .SH NAME
 slapd \- Stand-alone LDAP Daemon
 .SH SYNOPSIS
-.B LIBEXECDIR/slapd 
+.B slapd
 [\c
 .BR \-V [ V [ V ]]
 [\c
@@ -332,7 +332,7 @@ the LDAP databases defined in the default config file, just type:
 .LP
 .nf
 .ft tt
-	LIBEXECDIR/slapd
+	slapd
 .ft
 .fi
 .LP
@@ -343,7 +343,7 @@ on voluminous debugging which will be printed on standard error, type:
 .LP
 .nf
 .ft tt
-	LIBEXECDIR/slapd \-f /var/tmp/slapd.conf \-d 255
+	slapd -f /var/tmp/slapd.conf -d 255
 .ft
 .fi
 .LP
@@ -351,7 +351,7 @@ To test whether the configuration file is correct or not, type:
 .LP
 .nf
 .ft tt
-	LIBEXECDIR/slapd \-Tt
+	slapd -Tt
 .ft
 .fi
 .LP
-- 
2.37.1

